identity/client/src/pages/login/styled.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import styled from "styled-components";
+import { Stack } from "@carbon/react";
+import { spacing04, spacing05, spacing07 } from "@carbon/elements";
+import Page from "src/components/layout/Page.tsx";
+
+const loginPageContentWidth = "25rem";
+
+export const LoginFormContainer = styled(Stack)<{ $hasError: boolean }>`
+  flex-direction: column;
+  gap: ${spacing07};
+  width: 100%;
+  padding-top: ${({ $hasError }) => ($hasError ? 0 : spacing07)};
+  padding-bottom: ${spacing07};
+
+  > button {
+    max-inline-size: unset;
+    width: 100%;
+  }
+`;
+
+export const LoginPageContainer = styled(Page)`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+`;
+
+export const Content = styled.div`
+  width: ${loginPageContentWidth};
+  display: flex;
+  flex-direction: column;
+`;
+
+export const Header = styled.div`
+  margin: ${spacing07};
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  h1 {
+    font-size: ${spacing07};
+  }
+
+  img {
+    height: ${spacing07};
+  }
+`;
+
+export const LicenseInfo = styled.div`
+  width: 100%;
+  font-size: ${spacing04};
+  font-weight: 400;
+  text-align: center;
+  line-height: 1.3333;
+
+  a {
+    font-size: ${spacing04};
+  }
+`;
```

Comment:
If you use SCSS you can do use this util for [the font tokens](https://github.com/camunda/camunda/blob/main/tasklist/client/src/common/auth/Login/styles.module.scss#L43)

Reply:
@vsgoulart Do you mean it doesn't work with styled components?

Reply:
@vitorwtavares  The syntax `@include type.type-style('legal-01');` is SCSS only

I would actually recommend against adding more `styled-components` to our projects. I would like to start an initiative to remove it since it requires less strict CSP config which are important to us, it has some performance impact and it prevents us from using these SCSS utils Carbon provides us.

But since this initiative is not official yet I won't block this PR but you could at least use the vars like the screenshot below (which is what that Carbon util does)
![image](https://github.com/user-attachments/assets/ca6dfa3b-a17d-44e8-801e-d318d970dae8)